Dockerfile

```diff
@@ -1,29 +1,16 @@
 # Production build/deploy/start of DSDE Methods Repository Webservice
 
-# Basis image- Lukas K. recommended
-FROM debian:jessie
-MAINTAINER DSDE <dsde-engineering@broadinstitute.org>
+# http://github.com/broadinstitute/scala-baseimage
+FROM broadinstitute/scala-baseimage
 
-
-# Install necessary packages including java 8 jre and sbt and clean up apt caches
-RUN echo "deb http://ppa.launchpad.net/webupd8team/java/ubuntu trusty main" >> /etc/apt/sources.list.d/webupd8team-java.list && \
-    echo "deb-src http://ppa.launchpad.net/webupd8team/java/ubuntu trusty main" >> /etc/apt/sources.list.d/webupd8team-java.list && \
-    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys EEA14886 && \
-    echo debconf shared/accepted-oracle-license-v1-1 select true | /usr/bin/debconf-set-selections && \
-    echo debconf shared/accepted-oracle-license-v1-1 seen true | /usr/bin/debconf-set-selections
-
-RUN apt-get update && \
-    apt-get --no-install-recommends install -y --force-yes oracle-java8-installer &&\
-    apt-get clean autoclean && \
-    apt-get autoremove -y && \
-    rm -rf /var/lib/{apt,dpkg,cache,log}/ /var/cache/oracle-jdk8-installer 
-
```

Comment:
No need to run all the above commands anymore because the scala base image provides all of this.